master/static/migrations/20230725094922_add-rp-workspace-bindings-constraint.tx.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE rp_workspace_bindings
+    ADD CONSTRAINT rp_workspace_pkey PRIMARY KEY (workspace_id, pool_name);
```

Comment:
Nit: all lines should have the POSIX-compliant terminating `\n`, including the final lines of files.